frontend/app/components/Session_/StackEvents/StackEvents.js

```diff
@@ -1,85 +1,176 @@
+import { error as errorRoute } from 'App/routes';
+import JsonViewer from 'Components/Session_/StackEvents/UserEvent/JsonViewer';
+import Sentry from 'Components/Session_/StackEvents/UserEvent/Sentry';
+import { hideHint } from 'Duck/components/player';
+import withEnumToggle from 'HOCs/withEnumToggle';
+import { connectPlayer, jump } from 'Player';
 import React from 'react';
 import { connect } from 'react-redux';
-import { connectPlayer, jump } from 'Player';
-import { NoContent, Tabs } from 'UI';
-import withEnumToggle from 'HOCs/withEnumToggle';
-import { hideHint } from 'Duck/components/player';
-import { typeList } from 'Types/session/stackEvent'; 
-import UserEvent from './UserEvent';
+import { DATADOG, SENTRY, STACKDRIVER, typeList } from 'Types/session/stackEvent';
+import { NoContent, SlideModal, Tabs, Link } from 'UI';
 import Autoscroll from '../Autoscroll';
 import BottomBlock from '../BottomBlock';
+import UserEvent from './UserEvent';
 
 const ALL = 'ALL';
 
-const TABS = [ ALL, ...typeList ].map(tab =>({ text: tab, key: tab }));
+const TABS = [ALL, ...typeList].map((tab) => ({ text: tab, key: tab }));
 
 @withEnumToggle('activeTab', 'setActiveTab', ALL)
-@connectPlayer(state => ({
+@connectPlayer((state) => ({
   stackEvents: state.stackList,
+  stackEventsNow: state.stackListNow,
```

Comment:
I could, but it wouldn't change anything since it'd still filter (and then render) any time something changed in the store.

I think the selector solution is probably still most optimal, but it's a bit beyond the scope of this PR.